tools/docker/builder/Dockerfile

```
@@ -23,5 +23,11 @@ RUN apt-get update && apt-get install -y \
 	vim \
     && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /root/prplMesh
-ENTRYPOINT ["/root/prplMesh/tools/maptools.py", "build"]
+ARG uid
+ARG user
+ARG topdir
+RUN useradd -d /home/$user -m -s /bin/bash $user -u $uid
+
+USER $user
```

Doesn't the USER directive imply the useradd? I.e. isn't the useradd redundant?

You need to specify the UID of course, but that can be done by specifying the UID instead of the username.
```suggestion
USER $uid
```
You don't actually need a home directory for the user nor a shell.

---

ok